hoomd/filter/__init__.py

```diff
@@ -12,10 +12,10 @@
 to the same group *and the originally selected particles*, **even if the state
```

Comment:
The first line of this paragraph has a typo, it should read "particles" not "particle"

Reply:
Fixed.